src/v3/matchers.ts

```diff
@@ -537,25 +495,38 @@ export const matcherValueOrString = (obj: unknown): string => {
  * Recurse the object removing any underlying matching guff, returning the raw
  * example content.
  */
-export function reify(input: AnyTemplate): AnyJson {
+export function reify(input: unknown): AnyJson {
   if (isMatcher(input)) {
-    return reify(input.value as AnyTemplate);
+    return reify(input.value);
   }
 
-  if (Object.prototype.toString.call(input) === '[object Array]') {
```

Comment:
what on earth was going on here in the first place 😬

Reply:
Old school JS

Reply:
I think Array.isArray is “reasonably” new